### PR TITLE
Add message that cheatsheets are auto-generated

### LINKED ISSUE #369: Make it clear that cheetsheets are auto-generated
**Is your feature request related to a problem? Please describe.**
Sometimes somebody will raise a PR making manual changes to a cheatsheet file. These cheatsheet files are actually auto-generated and depend on files in pkg/i18n.

In [lazygit](https://github.com/jesseduffield/lazygit/blob/master/pkg/cheatsheet/generate.go#L71) we print a message at the top of the cheatsheet saying that it's auto-generated i.e.:
```go
		content = fmt.Sprintf("_This file is auto-generated. To update, make the changes in the "+
			"pkg/i18n directory and then run `%s` from the project root._\n\n%s", CommandToRun(), content)
```

We should do that here too in https://github.com/jesseduffield/lazydocker/blob/master/scripts/generate_cheatsheet.go

### PR DESCRIPTION
**Pull request by gusandrioli**
This PR resolves #369 

## Changes

Add message on top of each language's file specifying that cheatsheets are generated.

### CODE REVIEW

File: scripts/generate_cheatsheet.go [hunk 1]
```diff
@@ -40,6 +40,12 @@ func main() {
 
 		bindingSections := getBindingSections(mApp)
 		content := formatSections(mApp, bindingSections)
+		content = fmt.Sprintf(
+			"_This file is auto-generated. To update, make the changes in the "+
+				"pkg/i18n directory and then run `%s` from the project root._\n\n%s",
+			commandToRun(),
+			content,
+		)
```
Comment (gusandrioli) [inline scripts/generate_cheatsheet.go:48]: I followed `lazygit`'s pattern, but probably a slightly cleaner solution would be to convert `go run scripts/generate_cheatsheet.go` into a `const`. Let me know if you agree and i can update the PR.

**Comment by gusandrioli** [comment]
@jesseduffield I noticed some extra changes on the keybindings files, let me know if it's something version-wise that i'm missing.

File: scripts/generate_cheatsheet.go (hunk 1 above)
Comment (jesseduffield) [reply scripts/generate_cheatsheet.go:48]: @gusandrioli I'm happy for that to be converted to a const

**Comment by jesseduffield** [comment]
@gusandrioli those extra changes are likely intentional, so we're all good there

File: scripts/generate_cheatsheet.go (hunk 1 above)
Comment (gusandrioli) [reply scripts/generate_cheatsheet.go]: Updated!

**Comment by gusandrioli** [comment]
@jesseduffield Awesome, thanks. PR updated.

**Comment by jesseduffield** [comment]
great work @gusandrioli !

**Pull request merged**